### PR TITLE
update release notes and autodoc

### PR DESCRIPTION
**Pull request by whoburg**
@bqpd, ready for review.

### CODE REVIEW

**Comment by galbramc** [comment]
test this please

Windoze had a lingering process preventing a file from being deleted. I rebooted the VM to fix it.

**Comment by whoburg** [comment]
@bqpd, ping

File: docs/source/releasenotes.rst [hunk 1]
```diff
@@ -3,13 +3,48 @@ Release Notes
...
+Version 0.3.2
+-------------
+ * Modular / model composition fixes and improvements
+ * Working controlpanel() for Model
+ * ipynb and numpy dependency fixes
+ * printing fixes
+ * El Capitan fix
+ * slider widgets now have units
+
+Version 0.3.2
```
Comment (bqpd) [inline docs/source/releasenotes.rst]: @whoburg, should this be `Version 0.3.1`?

Comment (whoburg) [inline docs/source/releasenotes.rst]: good catch. this one is correct, but the one above should be 0.3.4. Will push a fix now.

**Comment by whoburg** [comment]
@bqpd, ready for re-review.

**Pull request merged**